From c86e2bb85075345fadaaf509e1382ed95bcdea19 Mon Sep 17 00:00:00 2001
Subject: [PATCH 09/12] Don't malloc(0).

This is covscan noticing that we're starting at 0 hunks instead of 1
hunk, and that means the first allocation is of 0 bytes, which is inane.

Signed-off-by: Peter Jones <pjones@redhat.com>
Signed-off-by: Curt Brune <curt@cumulusnetworks.com>

diff --git a/src/lib/efi.c b/src/lib/efi.c
index d19c00d..1b590ab 100644
--- a/src/lib/efi.c
+++ b/src/lib/efi.c
@@ -880,7 +880,7 @@ append_extra_args_file(uint8_t **data, size_t *data_size)
 	int fd = STDIN_FILENO;
 	ssize_t num_read=0;
 	unsigned long appended=0;
-	size_t maxchars = 0;
+	size_t maxchars = 1024;
 	char *buffer;
 
 	if (!data || *data) {
